pallets/order-book/src/tests/test_utils.rs

```diff
@@ -328,3 +328,20 @@ pub fn run_to_block(n: u32) -> Weight {
     }
     total_init_weight
 }
+
+pub fn update_orderbook_unchecked(
+    order_book_id: OrderBookId<AssetIdOf<Runtime>, DEXId>,
+    tick_size: Balance,
+    step_lot_size: Balance,
+    min_lot_size: Balance,
+    max_lot_size: Balance,
+) -> OrderBook<Runtime> {
+    let mut order_book = OrderBookPallet::order_books(order_book_id).unwrap();
+    order_book.tick_size.set(tick_size);
+    order_book.step_lot_size.set(step_lot_size);
```

Comment:
this doesn't perform any alignments, correct?

Reply:
just want to make sure it's intended

Reply:
Yes, without alignment. Just set raw values without any checks. It's necessary to make the preconditions setup more easy for some tests.